### DISCUSSION TITLE
Dark Theme issues on mobile after 255 Update

### DISCUSSION THREAD

**Discussion by Qstick** [requarks/wiki#4774] [Error / Bug Report] [answered]
After we updated from 201 to 255, dark theme seems borked on mobile devices

Below from Chrome on IPad 

![1265EE02-94AC-4087-B25F-28E09208C039](https://user-images.githubusercontent.com/376117/145684398-7611f72c-0359-4ddb-8c9a-ab9b72a5fda3.jpeg)

**Comment by NGPixel** [comment]
Do you have any custom CSS enabled on your wiki? Everything looks fine on my side...

<img width="770" alt="image" src="https://user-images.githubusercontent.com/15522395/146657678-813b83d8-90d3-4b56-9585-3e5772ce4dfd.png">

**Comment by Qstick** [comment]
Negative on the CSS override. Appears to only be a problem on Chrome mobile, confirmed by multiple people on multiple devices. Another screenshot from admin area. We updated to 260 today, same issue.

![FEA65403-A5B1-4BFA-B812-3A33D0C129D5](https://user-images.githubusercontent.com/376117/146661922-2ae2f9fe-1f26-463e-81f6-790cf650dd2f.jpeg)

wiki.servarr.com

**Comment by Qstick** [comment, marked as answer]
This was on our side with the proxy we use to redirect some wiki items...
